README.md

```diff
@@ -1,6 +1,7 @@
 # NCATS AWS AMIs 📀 #
```

Comment:
Might as well get rid of this old NCATS reference while we are here (I didn't search for any others in this repo; feel free to check for them if you want).

```suggestion
# CISA Cyber Hygiene AMIs 📀 #
```

Reply:
Please see [97f9e82](https://github.com/cisagov/cyhy_amis/pull/588/commits/97f9e828968fd526d67423b97ce674a12ef4b151).